templates/zerver/help/send-a-stream-message.md

```diff
@@ -1,31 +1,35 @@
 ## Sending a new stream message
 
-![New stream message](/static/images/help/new-stream-message.png)
+This guide is to teach you how to send a message to a new stream.
 
-To submit a new stream message, first click the **New stream message**
-button located at the bottom of your screen.
+  ![New stream message](/static/images/help/new-stream-message.png)
```

Comment:
Images should always appear after they're mentioned, per the doc styling guidelines.